README.md

```diff
@@ -133,18 +134,40 @@ postgres=# \dt
     - Code: 401 UNAUTHORIZED
 
         Content: `{ error : "You are unauthorized to make this request." }`
+### Alternate Auth
```

Comment:
with the disclaimer, naming is a tough thing :)

Should we call that simple auth? cc @idoqo

Reply:
"simple" makes sense to me too.